user_guide_src/source/extending/basecontroller/002.php

```diff
@@ -1,3 +1,8 @@
 <?php
 
-protected $helpers = ['html', 'text'];
+namespace App\Controllers;
+
+class Home extends BaseController
+{
+    protected $helpers = ['html', 'text'];
+}
```

Comment:
This is how to modify BaseController.

```suggestion
namespace App\Controllers;

// ...

abstract class BaseController extends Controller
{
    // ...
    
    protected $helpers = ['html', 'text'];
    
    // ...
}
```